### PR TITLE
Fixes obs group issue

### PR DESCRIPTION
**Pull request by areklalo**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java [hunk 1]
```diff
@@ -449,17 +448,47 @@ private void setObsValue(String valueText, Obs medicationHistoryObs) {
...
+			//Before we can crate obs group we have to save new Obs
+			medicationHistoryObs.setValueText("This observation represent obs group");	//this is added to avoid error.noValue
+			Context.getObsService().saveObs(medicationHistoryObs,null);
+		    String[] members = value.split(" ");
+		    for (String accessionNumber : members) {
+				Obs obs = getMatchedObs(accessionNumber);
+				if (obs != null) {
+					medicationHistoryObs.addGroupMember(obs);
+				}
+            }
```
Comment (pgesek) [inline api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java]: Fix identation

File: api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java [hunk 2]
```diff
@@ -449,17 +448,47 @@ private void setObsValue(String valueText, Obs medicationHistoryObs) {
 			}
 		} else if (valueText.contains("value-text")) {
 			medicationHistoryObs.setValueText(value);
-		}
+		} else if (valueText.contains("group-members")) {
+			//Before we can crate obs group we have to save new Obs
+			medicationHistoryObs.setValueText("This observation represent obs group");	//this is added to avoid error.noValue
```
Comment (pgesek) [inline api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java]: Fix identation

**Review by pgesek — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java (hunk 1 above)
Comment (areklalo) [reply api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java]: Changed

File: api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java (hunk 2 above)
Comment (areklalo) [reply api/src/main/java/org/openmrs/module/shr/cdahandler/processor/util/OpenmrsDataUtil.java]: Changed

**Review by pgesek — APPROVED** [review]

**Pull request merged**